DiamondLightSource/python-artemis
# 779 fix fluo optimise

Fixes #779 

Link to dodal PR (if required): [#114](https://github.com/DiamondLightSource/dodal/pull/114)

### To test:
1. Check issue requirements are all addressed
2. Confirm new tests run and are sensible
